projects/plugins/jetpack/_inc/client/components/navigation/index.jsx

```diff
@@ -43,13 +54,38 @@ export class Navigation extends React.Component {
 	};
 
 	trackRecommendationsClick = () => {
-		this.trackNavClick( 'recommendations' );
+		const isBubbleVisible = this.props.newRecommendationsCount > 0;
+
+		let recommendationsTrackData = {
+			target: 'nav_item',
+			path: 'recommendations',
+			is_new_recommendations_bubble_visible: isBubbleVisible,
+		};
+
+		if ( isBubbleVisible ) {
+			recommendationsTrackData = {
+				...recommendationsTrackData,
+				new_recommendations: this.props.newRecommendations ?? 'unknown',
```

Comment:
After our slack huddle, this doesn't really matter anymore, but my thought process was that "unknown" would be an accurate descriptor if there are new Recommendations but the `newRecommendations` array was empty. Though I guess `null` is a more standard way of defining that. I don't know, no need for more thought at this time since we are removing this 😆